etc/.golangci.yaml

```diff
@@ -91,7 +91,6 @@ linters-settings:
           - allowTypesBefore: "testing.TB,*testing.T,*testing.B,*testing.F"
       - name: context-keys-type
       - name: dot-imports
-      - name: empty-block
```

Comment:
It said my for-loop to drain the solution channel was empty and therefore "did nothing and could be removed".